test/integration/app/spec_build.js

```diff
@@ -345,4 +346,45 @@ describe('Spec option and build integration tests for app generator', function (
       assert.file('.build/debug/todo');
     });
   });
+
+  describe('Web application with an alert notification service is able to build', function () {
```

Comment:
Minor: description says "Web application" but spec is for a "Basic application". The difference doesn't seem significant for the test. Either change the description or alter the spec. This is minor and shouldn't block merging.